include/odp/api/spec/queue.h

```diff
@@ -393,10 +393,11 @@ odp_schedule_group_t odp_queue_sched_group(odp_queue_t queue);
  *
  * @param queue   Queue handle
  *
- * @return Number of ordered locks associated with this ordered queue
- * @retval <0 Specified queue is not ordered
+ * @return	Number of ordered locks associated with this ordered queue
+ * @retval 0	Specified queue is not ordered or No ordered lock associated
+ *		with the ordered queue.
```

Comment:
Typo: upper case "No" in the middle of sentence -> change to lower case "no"

Reply:
Okay